mlir_graphblas/src/lib/GraphBLAS/GraphBLASPasses.cpp

```diff
@@ -276,6 +278,8 @@ struct MatrixSelectOutputWriter {
 
     callResizeIndex(rewriter, module, loc, tensor, c1, nnz);
     callResizeValues(rewriter, module, loc, tensor, nnz);
+
+    tensor = convertToExternalCSR(rewriter, module, loc, tensor);
```

Comment:
I think this is redundant. I'll change this to an assert since `createTensor` already makes sure it's a CSR tensor.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/1242e169902c7829e5eda1f3b7d772d60782a348 accomplishes this.